### PR TITLE
fixing migration 4.x guide

### PR DESCRIPTION
**Pull request by tylerkelly13**
Fixing the migration from 4.x guide using bug info in the 5.0 release notes and fixing the 5.0 release notes

### CODE REVIEW

File: _includes/release-notes/new-features.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ The following new features were added to the TinyMCE 5.0 RC version.
...
 
 ### Context menu
 
-The [Context menu](https://www.tiny.cloud/docs/plugins/contextmenu/) is no longer a plugin, it is part of the core and always enabled. Where TinyMCE 4.x only supported adding registered menu items, the new context menu also allows plugins to register "sections" of the context menu. These sections are dynamic and may show or hide depending on the cursor position when the context menu is opened. For example, the default context menu config is now `'link image editimage table spelling'` which are all plugin section references, not menu items. 
+The [Context menu](https://www.tiny.cloud/docs/plugins/contextmenu/) is no longer a plugin, it is part of the core and always enabled. Where TinyMCE 4.x only supported adding registered menu items, the new context menu also allows plugins to register "sections" of the context menu. These sections are dynamic and may show or hide depending on the cursor position when the context menu is opened. For example, the default context menu config is now `'link image editimage table spelling'` which are all plugin section references, not menu items.
```
Comment (lnewson) [inline _includes/release-notes/new-features.md]: Should this use v4 link? As the content of this link doesn't seem very useful.

File: migration-from-4x.md [hunk 2]
```diff
@@ -542,9 +524,27 @@ These features have either changed or have been deleted in TinyMCE 5.0.
 
 | **Plugin name** | **Description** |
 | --------------- |  -------------- |
-| [ContextMenu](https://www.tiny.cloud/docs/plugins/contextmenu/) | New API. See [docs]({{site.baseurl}}/ui-components/contextmenu/). |
-| [ColorPicker](https://www.tiny.cloud/docs/plugins/colorpicker/) | Moved to the core. See [docs]({{site.baseurl}}/configure/content-appearance/#color_picker}}). |
-| [Text Color](https://www.tiny.cloud/docs/plugins/textcolor/) | The `textcolor` plugin was removed and this setting is not required in TinyMCE 5.0. |
+| [ContextMenu]({{site.baseurl}}/plugins/contextmenu/) | New API. See [docs]({{site.baseurl}}/ui-components/contextmenu/). |
```
Comment (lnewson) [inline migration-from-4x.md]: I think all these should be v4 links as well, since it's linking to what was removed?

File: migration-from-4x.md [hunk 3]
```diff
@@ -542,9 +524,27 @@ These features have either changed or have been deleted in TinyMCE 5.0.
...
+
+#### Custom context toolbars
+
+The Context Toolbar accepts toolbar buttons that were previously added to the editor using the editor's `addButton`, `addToggleButton`, `addSplitButton` or a`ddMenuButton` functions. The method for creating custom context toolbars in TinyMCE 5.0 has also been changed from `editor.addContextToolbar()` to `editor.ui.registry.addContextToolbar()`.
+
+For information on Context Toolbars, see the [docs]({{site.baseurl}}/ui-components/contexttoolbar).
+
+#### Custom context menus
+
+The [Context menu]({{site.baseurl}}/plugins/contextmenu/) is no longer a plugin, it is part of the core and always enabled. Where TinyMCE 4.x only supported adding registered menu items, the new context menu also allows plugins to register "sections" of the context menu. These sections are dynamic and may show or hide depending on the cursor position when the context menu is opened. For example, the default context menu config is now `'link image editimage table spelling'` which are all plugin section references, not menu items. For information on using context menus, see: [UI components - Context menu]({{site.baseurl}}/ui-components/contextmenu/).
```
Comment (lnewson) [inline migration-from-4x.md]: One more that should maybe be a v4 link?

File: _includes/release-notes/new-features.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/release-notes/new-features.md]: fixed

File: migration-from-4x.md (hunk 3 above)
Comment (tylerkelly13) [reply migration-from-4x.md]: Fixed

File: migration-from-4x.md (hunk 2 above)
Comment (tylerkelly13) [reply migration-from-4x.md]: Fixed

**Pull request merged**